Makefile

```diff
@@ -12,7 +12,11 @@ CMD_GITHUB ?= gh
 # environment:
 ARCH ?= $(shell uname -m)
 KERN_RELEASE ?= $(shell uname -r)
-KERN_SRC ?= /lib/modules/$(KERN_RELEASE)/build
+KERN_BLD_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/build)
+KERN_SRC_PATH ?= $(if $(KERN_SRC),$(KERN_SRC),/lib/modules/$(KERN_RELEASE)/source)
+ifeq ($(wildcard $(KERN_SRC_PATH)/.),)
```

Comment:
maybe instead of enumerating all the files just `test -d`?

Reply:
I understand that this line checks if the directory exists, isn't it?
I'll try with "test -d" if this is better